pr-dhl-woocommerce/includes/class-pr-dhl-wc-order-paket.php

```diff
@@ -549,6 +550,23 @@ public function crossborder_and_domestic_fields( $dhl_label_items, $is_disabled
 				)
 			);
 
+			$order             = wc_get_order( $order_id );
+			$base_country_code = PR_DHL()->get_base_country();
+
+			if ( ( $base_country_code == 'DE' ) && ( $this->is_shipping_domestic( $order_id ) ) ) {
```

Comment:
You can add GoGreen option here instead instead of duplicating the checks 
https://github.com/shadimanna/dhl-logistic-services-for-woocommerce/blob/2f7b2c6ebaf93f2b71726da343953ba2c1483677/pr-dhl-woocommerce/includes/class-pr-dhl-wc-order-paket.php#L72